neuro_flow/live_runner.py

```diff
@@ -330,7 +331,10 @@ async def run(
         if job.title:
             run_args.append(f"--description={job.title}")
         if job.name:
-            run_args.append(f"--name={job.name}")
+            name = job.name
+        else:
+            name = f"{self._flow.project.id}--{job_id}".replace("_", "-")
```

Comment:
I think we should include a multi-job suffix for multi jobs, or users will have to do it manually because the platform forbids jobs with the same names.